build.gradle

```diff
@@ -97,3 +95,8 @@ idea.project.ipr.withXml { xml ->
 jdks {
     daemonTarget = 17
 }
+
+javaVersions {
+    libraryTarget = 21
```

Comment:
Will need to test how this works in gradle too. I suspect while we still support Java 17 in our gradle plugins we're going to have to do something a bit cleverer here too. This may also be the impetus to get us off spotless (or just upgrade every project to use 21 runtime).